client/app/services/custom-button/custom-button.html

```diff
@@ -23,12 +23,13 @@
     </button>
     <ul uib-dropdown-menu role="menu" aria-labelledby="{{buttonGroup.id}}">
       <li ng-repeat="button in buttonGroup.buttons" role="presentation">
-        <a class="custom-button-action"
+        <a class="custom-button-action" 
            role="menuitem"
            tabindex="-1"
+           ng-class="{'disabled': !button.enabled}"
```

Comment:
this is all ya should need to conditionally apply disabled style, unsure about rolling your own class

Reply:
We didn't have any classes that matched the disabled styling that is traditionally done in our other drop downs.  Without rewriting the button component to match exactly the UI-bootstrap drop downs, this was the easiest path to a solution.

Reply:
🤔 but we do have disabled buttons, does applying the disabled status alone yield the disabled appearance?

Reply:
This is actually for drop downs not "buttons" per se.  We dont have any classes that handled disabling menu options in a drop down.  The html is actually A tags so some styling had to be done to simulate a disabled state since A tags don't natively have a disabled state.  I made the styling match how disabled seemed to look and operate on uib-dropdown elements.

Reply:
👍 can we get a ss of the change in action?

Reply:
Ok last ask then i'll approve... still super suspicious about that color being used that isn't defined in the color file being imported, would you mind building the app, checking from /ui/service? Promise after that i'll go away